zeebe/gateway-protocol/src/main/proto/gateway.proto

```diff
@@ -112,6 +112,34 @@ message JobResult{
   // As a result, the completion request is rejected and the task remains active.
   // Defaults to false.
   optional bool denied = 1;
+  // JSON object with attributes that were corrected by the worker.
+  // The following attributes can be corrected, additional attributes will be ignored:
+  //   * `assignee` - reset by providing an empty String
+  //   * `dueDate` - reset by providing an empty String
+  //   * `followUpDate` - reset by providing an empty String
+  //   * `candidateGroups` - reset by providing an empty list
+  //   * `candidateUsers` - reset by providing an empty list
+  //   * `priority` - minimum 0, maximum 100, default 50
+  optional JobResultCorrections corrections = 2;
+}
+
+message JobResultCorrections {
+  // The assignee of the task.
+  optional string assignee = 1;
+  // The due date of the task.
+  optional string dueDate = 2;
+  // The follow-up date of the task.
+  optional string followUpDate = 3;
+  // The list of candidate users of the task.
+  optional StringList candidateUsersList = 4;
+  // The list of candidate groups of the task.
+  optional StringList candidateGroupsList = 5;
```

Comment:
🔧 We could consider aligning the naming here with the REST API:
```suggestion
  // The list of candidate users of the task.
  optional StringList candidateUsers = 4;
  // The list of candidate groups of the task.
  optional StringList candidateGroups = 5;
```

Reply:
I will adjust the naming, very good point 👍